ocictl/cmd/build.go

```diff
@@ -70,7 +70,16 @@ func (b *buildCmd) run(args []string) error {
 		return err
 	}
 
-	switch v1alpha1.Framework(b.builder) {
+	// Prioritise builder passed in as argument, default builder is docker
+	builder := b.builder
```

Comment:
shouldn't this be simply as  if the default is docker

```
builder := b.builder
if !ociBuilderSpec.Daemon {
			builder = "buildah"
		}
```

Reply:
I feel like I had a reason for this or maybe I wasn't thinking straight - will make this change